### DISCUSSION TITLE
Example Configuration for Future3 Battery monitoring

### DISCUSSION THREAD

**Discussion by saeugetier** [MiczFlor/RPi-Jukebox-RFID#2025] [Q&A] [answered]
I cannot find any example or documentation for the newly introduced battery monitoring in future3. How can I enable the battery monitor in the yaml configuration?

**Comment by pabera** [comment]
@arne123 Arne built this feature. Maybe they get the chance to answer here.

**Comment by pabera** [comment]
I remembered that we had some documentation about it but it took me some time to find it. Have a look at the following file's comments. You will find some useful information there. https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/src/jukebox/components/battery_monitor/batt_mon_i2c_ads1015/__init__.py

If you get this to work, let us know. It would be great to have this in the general documentation as well. Maybe this is an opportunity to contribute ;-)

**Comment by arne123** [comment, marked as answer]
at that time I put something like this into the config, not sure it will work with the actual version though

modules:
  named:
    ......
    battmon: battery_monitor.batt_mon_i2c_ads1015


battmon:
  scale_to_phy_num: 35
  scale_to_phy_denom: 15
  warning_action:
  all_clear_action:
